src/components/academy/NavigationBar.tsx

```diff
@@ -24,6 +31,12 @@ const NavigationBar: React.SFC<NavigationBarProps> = props => (
       >
         <Icon icon={IconNames.FLAME} />
         <div className="navbar-button-text hidden-xs">Missions</div>
+        <NotificationBadge
```

Comment:
You should consider exporting a simple badge in your notifications file that just displays a number. Or you could just put `<Tag></Tag>` here since all we want is the number. 

I find it pointless that your:
- filtering and passing in all your notifications to the component
- the component is running a map on your component and processing them

And eventually they wont even be shown since `enableHover` is false?

Reply:
Noted. I will disable processing of the tags if enableHover is false